### DISCUSSION TITLE
[dcc64 Error] FB4D.Document.pas(1458): E2003 Undeclared identifier: 'DataType'

### DISCUSSION THREAD

**Discussion by johnbh3** [SchneiderInfosystems/FB4D#167] [General]
Erro [dcc64 Error] FB4D.Document.pas(1458): E2003 Undeclared identifier: 'DataType' unit FB4D.Document com delphi alexandria.

**Comment by SchneiderInfosystems** [comment]
Why do you have closed this issue? Is this issue still unsolved in the published Version of 1.6? I think so - see discussion #169.
